specs/disciplina/witness/api/witness.yaml

```diff
@@ -368,6 +368,22 @@ paths:
       requestBody:
         $ref: '#/components/requestBodies/FairCV'
 
+  /checkcv-pdf:
+    put:
+      summary: Check the validity of FairCV embedded in PDF document.
+      operationId: checkFairCVPDF
+      responses:
+        200:
+          description: >-
+            FairCV in a PDF was well-formed and has been successfully checked. The
+            result of checking is returned in a response along with FairCV itself.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FairCVAndCheckResult'
+      requestBody:
+        $ref: '#/components/requestBodies/ByteString'
```

Comment:
You forgot to add the request body specification to `#/components/requestBodies`, which makes the spec invalid (editor.swagger.io will complain if you try to upload it). Also, this is not generic `ByteString`, it should be called `PDF`, because request content type should be `application/pdf`, not `application/octet-stream`

Reply:
> `#/components/requestBodies`

Is this a local address in the same yaml document?

Reply:
@Heimdell yes, the section where request bodies are defined is literally a couple of lines lower.